ansible/roles/haproxy/templates/haproxy.cfg.j2

```diff
@@ -1,23 +1,59 @@
-####################
-# api-proxy config #
-####################
+{{ ansible_managed | comment }}
 
-frontend front-api-proxy
-    bind {{ keepalived_vip }}:8443
-    bind 127.0.0.1:8443
+# This configuration provides some sensible defaults - modeled on the default configuration found here
+# https://github.com/haproxytech/haproxy-docker-alpine/blob/master/2.3/haproxy.cfg
+global
+    log stdout format raw local0
+    stats   socket    /var/run/haproxy.stat
+    user    haproxy
+    group   haproxy
+    chroot  /var/empty
+    daemon
+
+defaults
+    mode                    http
+    log                     global
+    option                  httplog
+    option                  dontlognull
+    option                  http-server-close
+    option                  redispatch
+    retries                 5
+    timeout http-request    5m
+    timeout queue           5m
+    timeout connect         30s
+    timeout client          {{ haproxy_keepalive_timeout }}
+    timeout server          15m
+    timeout http-keep-alive 30s
+    timeout check           30s
+    maxconn                 4000
+
+frontend stats
+    bind *:8404
+    mode  http
+    stats enable
+    stats uri /stats
+    stats refresh 10s
+    stats admin if TRUE
+
+{% if haproxy_healthcheck_port is defined -%}
+frontend healthz
+    bind *:{{ haproxy_healthcheck_port }}
+    mode http
+    monitor-uri /healthz
+{% endif -%}
+
+frontend apiserver
+    bind {{ cluster_apiserver_frontend_ip }}:{{ cluster_apiserver_frontend_port }}
```

Comment:
`cluster_apiserver_frontend_ip` isn't being initialized in defaults or the group vars